mayastor-test/test_nexus.js

```diff
@@ -343,30 +346,65 @@ describe('nexus', function() {
     });
   });
 
+  /*  FIXME:
+    Unfortunately some 3rd party javascript library in the
+    test framework changes the invalid value of 100
+    to a valid value of 0.
+  it('should fail to publish with invalid ShareProtocol value of 100', done => {
+      let args = {
+          uuid: UUID,
+          share: 100,
+      };
+
+      client.PublishNexus(args, (err, data) => {
+          if (!err) return done(new Error('Expected error'));
+          assert.equal(err.code, grpc.status.INVALID_ARGUMENT);
+          done();
+      });
+  });
+*/
+
   it('should publish the nexus using nbd', done => {
-    client.PublishNexus({ uuid: UUID }, (err, res) => {
-      assert(res.device_path);
-      nbd_device = res.device_path;
-      done();
-    });
+    // TODO: repeat this test for Nvmf
```

Comment:
Todo...

Reply:
There is already a Jira (CAS-26) for Nvmf integration testing. I'll remove the TODO comments.